src/amo/components/Home/index.js

```diff
@@ -170,6 +170,9 @@ export class HomeBase extends React.Component {
 
     return (
       <div className="Home">
+        {/* IT looks for this comment with nagios, don't remove it. */}
+        <span dangerouslySetInnerHTML={{ __html: '<!-- aww yeah -->' }} />
```

Comment:
So this should probably just be an HTML comment–won't that work? Or will react strip it out?

Also I wish we used a more fun string. This is what IT look for on addons-server so maybe ours should be different.

Maybe we could use "Godzilla of browsers" or something cute 😉

Reply:
> Or will react strip it out?

I am pretty sure React removes all comments.

Reply:
Ah okay I figured that might be the case. Cool then!

Reply:
Yes react removes all the comments.
And yes, writing test case is much better idea.
`Godzilla of browsers` \o/